nilearn/decoding/space_net.py

```diff
@@ -359,8 +358,6 @@ def path_scores(
         Indicates whether the loss is a classification loss or a
         regression loss.
 
-    Xmean: ??? TODO: Add description.
-
     key: ??? TODO: Add description.
```

Comment:
Maybe we could take this opportunity to add it ?

Reply:
probably in a separate PR